### PR TITLE
Pass through more linker options to llvm

### PR DESCRIPTION
**Pull request by devpow112**

### CODE REVIEW

**Review by AWoloszyn — APPROVED** [review]

**Review by AWoloszyn — APPROVED** [review]

File: Libraries/WNScripting/CMakeLists.txt [hunk 1]
```diff
@@ -74,22 +76,32 @@ if (NOT WN_DISABLE_CXX)
...
+      "-DCMAKE_JOB_POOLS=${JOB_POOLS} "
+      "-DCMAKE_JOB_POOL_LINK=${CMAKE_JOB_POOL_LINK} "
+      "-DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS} "
       "-DCMAKE_CXX_FLAGS_DEBUG=${CMAKE_CXX_FLAGS_DEBUG} "
       "-DCMAKE_CXX_FLAGS_RELEASE=${CMAKE_CXX_FLAGS_RELEASE} "
       "-DCMAKE_CXX_FLAGS_MINSIZEREL=${CMAKE_CXX_FLAGS_MINSIZEREL} "
       "-DCMAKE_CXX_FLAGS_RELWITHDEBINFO=${CMAKE_CXX_FLAGS_RELWITHDEBINFO} "
-      "-DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS} "
+      "-DCMAKE_CXX_COMPILER_LAUNCHER=${CMAKE_CXX_COMPILER_LAUNCHER} "
+      "-DCMAKE_C_FLAGS=${CMAKE_C_FLAGS} "
```
[the remark this answers is not in the document]
Comment (devpow112) [reply Libraries/WNScripting/CMakeLists.txt]: I'll leave it for now if that's ok

**Pull request merged**